cdap-ui/app/cdap/components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/index.tsx

```diff
@@ -14,48 +14,73 @@
  * the License.
  */
 
-import Divider from '@material-ui/core/Divider';
+import * as React from 'react';
+
 import withStyles, { StyleRules, WithStyles } from '@material-ui/core/styles/withStyles';
-import ExpandLiveViewButton from 'components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/ExpandLiveViewButtton';
+
+import Divider from '@material-ui/core/Divider';
 import JSONDownloadButton from 'components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/JSONDownloadButton';
 import JSONImportButton from 'components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/JSONImportButton';
-import * as React from 'react';
+import LiveConfigurationGroupButton from 'components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/LiveConfigurationGroupButton';
+import LiveJSONButton from 'components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/LiveJSONButton';
+import { LiveViewMode } from 'components/PluginJSONCreator/Create/PluginJSONMenu';
+import classnames from 'classnames';
 
 const styles = (theme): StyleRules => {
   return {
     JSONActionButtons: {
-      padding: '15px',
       flexDirection: 'column',
+      borderRight: `1px solid ${theme.palette.divider}`,
+      height: 'calc(100vh - 53px)', // height excluding footer
```

Comment:
what about header?

Reply:
It has to be height excluding header, since JSON action buttons are located in live viewer component.

Resolved.

<img width="1792" alt="Screen Shot 2020-08-04 at 10 45 10 AM" src="https://user-images.githubusercontent.com/14116152/89307859-92a82c00-d63f-11ea-8324-de2a6ab8918a.png">